docs/manual/scala/guide/cluster/code/ChildActors.scala

```diff
@@ -0,0 +1,124 @@
+package docs.scaladsl.cluster.childactors
+
+import akka.actor.{Actor, ActorSystem, Props}
+import akka.cluster.sharding.{ClusterSharding, ClusterShardingSettings, ShardRegion}
+import akka.testkit.{TestKit, TestProbe}
+import akka.util.Timeout
+import com.lightbend.lagom.scaladsl.persistence.PersistentEntity.ReplyType
+import com.lightbend.lagom.scaladsl.persistence.{ChildPersistentEntity, ChildPersistentEntityFactory, PersistentEntity}
+import org.scalatest.{BeforeAndAfterAll, WordSpecLike}
+
+//#create-child-persistent-entity-factory
+class MyProcessManagerRouter(system: ActorSystem, entityFactory: () => MyProcessManagerEntity) {
+  private val processManager = {
+    val factory = ChildPersistentEntityFactory.forEntity(entityFactory)
+    ClusterSharding(system).start(
+      "MyProcessManager",
+      Props(new MyProcessManager(factory)),
+      ClusterShardingSettings(system),
+      extractEntityId,
+      extractShardId
+    )
+  }
+  //#create-child-persistent-entity-factory
+
+  private def extractEntityId: ShardRegion.ExtractEntityId = PartialFunction.empty
+  private def extractShardId: ShardRegion.ExtractShardId = _ => ""
+}
+
+trait MyCommand
+
+case object StartProcess extends MyCommand with ReplyType[String]
+
+class MyProcessManagerEntity extends PersistentEntity {
+  override type Command = MyCommand
+  override type Event = String
+  override type State = String
+  override def initialState = ""
+  override def behavior = {
+    case "" => Actions()
+  }
+}
+
+//#create-child-persistent-entity
+class MyProcessManager(
+    factory: ChildPersistentEntityFactory[MyProcessManagerEntity]
+  ) extends Actor {
+
+  private var entity: ChildPersistentEntity[MyCommand] = _
+
+  override def preStart(): Unit = {
+    entity = factory(
+      context.self.path.name, // The entity id
+      "entity"                // Name of the child actor
```

Comment:
In the final docs it could be clearer if the name was something else rather than `"entity"`.

Reply:
`entity` is actually what I think is the most sensible name for the actor in most cases, in the real world cases where I've used this I've called it `entity`. Where the name will actually most likely appear to end users is when they are doing monitoring. When cluster sharding, the name of the actor will end up being `/user/{typeName}/{shardId}/{entityId}/entity`, so for a process with ID `c039449b-0ca0-4f91-bcca-310accaa3a91` which hashed to shard `23`, the actor path would be `/user/MyProcessManager/23/c039449b-0ca0-4f91-bcca-310accaa3a91/entity`. When you see this, you can instantly see that this actor is the child entity of the process with ID `c039449b-0ca0-4f91-bcca-310accaa3a91`. The only time I would name it any different is if I had multiple different child entities, and I needed to distinguish them from each other.

Reply:
Thanks @jroper. Do you think it'd make sense to include this paragraph in the docs?